test/Microsoft.TestPlatform.AcceptanceTests/LoggerTests.cs

```diff
@@ -192,7 +192,7 @@ private void IsFileAndContentEqual(string filePath)
         }
 
         string filePathContent = sb.ToString();
-        string[] divs = { "Total tests", "Passed", "Failed", "Skipped", "Run duration", "Pass percentage", "SampleUnitTestProject.UnitTest1.PassingTest" };
+        string[] divs = { "Total tests", "Passed", "Failed", "Skipped", "Run duration", "Pass percentage", "PassingTest" };
```

Comment:
Test method now has a display name so the html logger no longer writes the fully qualified name.